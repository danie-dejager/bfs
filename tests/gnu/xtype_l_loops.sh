bfs_diff loops -xtype l
