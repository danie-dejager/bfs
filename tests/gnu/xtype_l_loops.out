loops/broken
loops/loop
loops/notdir
